forge-game/src/main/java/forge/game/ability/ApiType.java

```diff
@@ -204,7 +204,9 @@ public enum ApiType {
     DamageResolve (DamageResolveEffect.class),
     ChangeZoneResolve (ChangeZoneResolveEffect.class),
     InternalLegendaryRule (CharmEffect.class),
-    InternalIgnoreEffect (CharmEffect.class);
+    InternalIgnoreEffect (CharmEffect.class),
```

Comment:
imo we should also get rid of ChooseCompanionEffect class and map it to Charm as placeholder like those others